grassroots-backend/test/Auth.e2e-spec.ts

```diff
@@ -1,25 +1,49 @@
-import { useE2ETestFixture } from "../src/testing/E2eSetup";
 import { describe, expect, it } from "vitest";
-import { AppModule } from "../src/App.module";
+import { useE2ETestFixture } from "../src/testing/E2eSetup";
```

Comment:
I probably spent more time trying to get tests to work than getting the functionality working.

This isn't great, but nailing this stuff is pretty subtle, and mostly just testing 3p libraries. 

Ideally we'd have a test that this does actually insert a user into the database. Happy to poke at it more if you think that's worthwhile, but I wanted to at least get this in front of you in the current state.